src/renderer/components/battle/Playerlist.vue

```diff
@@ -14,12 +14,32 @@ SPDX-License-Identifier: MIT
         :title="t('lobby.components.battle.addBotModal.title')"
         @bot-selected="onBotSelected"
     />
-    <div class="scroll-container padding-right-sm">
+    <div v-if="battleStore.isOnline" class="scroll-container padding-right-sm">
+        <div class="playerlist" :class="{ dragging: draggedBot || draggedPlayer }">
+            <TeamComponent
+                v-for="(team, key, index) in tachyonStore.activeLobby != undefined ? tachyonStore.activeLobby!.allyTeams : {}"
+                :key="key"
+                :teamId="index"
+                :teamKey="key as string"
```

Comment:
Once tachyon server sends us the allyteam/team with matching strings to the member's properties, we can probably remove this ``as``.

Reply:
Still required? This seems to be an issue where Vue treats the key as it will be ``string | number`` when we know it is always a string. Will keep the ``as`` here.